teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,28 @@
-try:
-    import pycodestyle as pep8
-except ImportError:
-    import pep8
+from flake8.formatting import base
 
 import re
 
 from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
```

Comment:
I can see that having to specify a CLI option is not desirable because folks may have a tox.ini that invokes flake8 and you want it to work in both dev envs and CI and use the best output format. In other words, ideally you'd automatically get TeamCity format when running in TeamCity but you don't want that while developing. 

How best to solve this?